tests/scripts/all.sh

```diff
@@ -557,6 +563,17 @@ cleanup
 cmake -D CMAKE_BUILD_TYPE:String=Check .
 if_build_succeeded tests/scripts/key-exchanges.pl
 
+if [ $TEST_CXX -ne 0 ]; then
+    msg "build: Unix make, gcc and g++ test" # ~ 30s
+    cleanup
+    make TEST_CPP=1
+
+    msg "build: cmake, gcc and g++ test" # ~ 30s
+    cleanup
+    CC=gcc cmake -D TEST_CPP=YES .
```

Comment:
Does `CC=gcc` influence the _C++_ compiler? Seems strange to me, I'd expect the C++ compiler to be `${CXX}`.

Reply:
It does not, It's just the usual syntax used for other tests in all.sh, and I thought it's a prerequisite to compile the usual sources (which are getting compiled during this test too).